ui/views_test.py

```diff
@@ -17,10 +18,11 @@ def setUp(self):
 
     def test_index_view(self):
```

Comment:
Just two more things:
 - Can you rename this test to `test_program_liveness`
 - In this test create another Program where live=False, and assert that the program title is not in the response